src/open-websocket.js

```diff
@@ -0,0 +1,14 @@
+import Html5WebSocket from 'html5-websocket'
+import ReconnectingWebSocket from 'reconnecting-websocket'
+import WebSocket from 'ws'
+
+export default wsAddressBuilder => {
+  return new ReconnectingWebSocket(wsAddressBuilder, undefined, {
```

Comment:
`wsAddressBuilder` will here be a function, but the library takes an url, what is the reason to not pass the url directly?